README.md

```diff
@@ -1,7 +1,331 @@
+
+client.PermissionManager.AddOrUpdateIdentity(expandedProviderId, null, userBody);
+client.PermissionManager.AddOrUpdateIdentity(expandedProviderId, null, groupBody);
+```
+
+### Push identities in batch
+If possible, you should push the identity in batch.
+```csharp
+PermissionIdentity member1 = new PermissionIdentity(@"acme\member1", PermissionIdentityType.User);
+PermissionIdentity member2 = new PermissionIdentity(@"acme\member2", PermissionIdentityType.User);
+
+PermissionIdentityBody bodyMember1 = new PermissionIdentityBody(member1);
+bodyMember1.Mappings.Add(new PermissionIdentity("member1@acme.com", PermissionIdentityType.User));
+
+PermissionIdentityBody bodyMember2 = new PermissionIdentityBody(member2);
+bodyMember2.Mappings.Add(new PermissionIdentity("member2@acme.com", PermissionIdentityType.User));
+
+List<PermissionIdentityBody> mappingsMembersToAddOrUpdate = new List<PermissionIdentityBody> {
+    bodyMember1,
+    bodyMember2
+};
+
+List<PermissionIdentityBody> membersToAddOrUpdate = new List<PermissionIdentityBody> {
+    new PermissionIdentityBody(new PermissionIdentity(@"acme\team2", PermissionIdentityType.Group)) {
+        Members = new List<PermissionIdentity> {
+            member1,
+            member2
+        }
+    }
+};
+
+// Push user mappings and group members in one batched operation.
+client.PermissionManager.AddOrUpdateIdentities(expandedProviderId, null, mappingsMembersToAddOrUpdate.Concat(membersToAddOrUpdate).ToList());
+```
+
+### Disable a single identity
+You can easily disable an identity. For more information, visit [Disabling a Single Security Identity](https://docs.coveo.com/en/84/cloud-v2-developers/disabling-a-single-security-identity).
+```csharp
+client.PermissionManager.DeleteIdentity(expandedProviderId, new PermissionIdentity(@"acme\johndoe", PermissionIdentityType.User));
+```
+
+### Disable identities in batch
+```csharp
+IList<PermissionIdentity> identitiesToDelete = new List<PermissionIdentity> {
+    new PermissionIdentity(@"acme\member1", PermissionIdentityType.User),
+    new PermissionIdentity(@"acme\member2", PermissionIdentityType.User)
+};
+
+client.PermissionManager.DeleteIdentities(expandedProviderId, null, identitiesToDelete);
+```
+
+### Disable identities older than a specific ordering ID
+Same as the documents, you can disable identities that have an ordering ID smaller than the one you provide
+```csharp
+client.PermissionManager.AddOrUpdateIdentity(expandedProviderId, 100, new PermissionIdentityBody(new PermissionIdentity(@"acme\tobedeleted3", PermissionIdentityType.User)));
+client.PermissionManager.AddOrUpdateIdentity(expandedProviderId, 200, new PermissionIdentityBody(new PermissionIdentity(@"acme\tobedeleted4", PermissionIdentityType.User)));
+
+// Wait a little bit, then disable all identities with an ordering ID less than 300.
+client.PermissionManager.DeleteIdentitiesOlderThan(expandedProviderId, 300);
+```
+
```

Comment:
Enlever la ligne ici ou en mettre avant tous les 'good to know'?

Reply:
Jcrois que un saut de ligne ici c'est pas représentatif du document, alors je vais l'enlever pour etre pareil partout!

```suggestion
```